WinUIGallery/WinUIGallery.csproj

```diff
@@ -83,6 +83,30 @@
         <AppxBundle>Never</AppxBundle>
     </PropertyGroup>
 
+    <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug-Unpackaged|x86'" />
```

Comment:
Are changes here intentional? If so, what is it for?

Reply:
> Are changes here intentional? If so, what is it for?

Fixed it.